packages/@uppy/webcam/src/RecordingLength.js

```diff
@@ -0,0 +1,12 @@
+const { h } = require('preact')
+const formatSeconds = require('./formatSeconds')
+
+module.exports = function RecordingLength ({ recordingLengthSeconds }) {
+  const formattedRecordingLengthSeconds = formatSeconds(recordingLengthSeconds)
+
+  return (
+    <div class="uppy-Webcam-recordingLength" title={formattedRecordingLengthSeconds} aria-label={formattedRecordingLengthSeconds}>
```

Comment:
Ideally `aria-label` should say `Recording length 1:25`. We need “Recording length: {number}” string added to Webcam’s `i18n`, please see: https://github.com/transloadit/uppy/blob/895498815718cf3ef4245593e176f0ccc6503fc3/packages/%40uppy/core/src/index.js#L53 and https://github.com/transloadit/uppy/blob/895498815718cf3ef4245593e176f0ccc6503fc3/packages/%40uppy/core/src/index.js#L830.

Title should probably just say “Recording length”, since the number is visually available. We could also omit the `title` altogether, because if we keep it, we’ll have to add 2 `i18n` strings — one with the number, other without the number.

Reply:
Thanks @arturi , I've made those changes and also added translations from Google Translate for DE, ES, FR, EN. Let me know if you want me to add all translations - I'm not sure on what the Uppy policy is on supplying translations as part of the PR - couldn't find it in the Contributing Readme. 

All tests pass locally but Travis and Netflify builds are failing - looks like other PRs are failing too for the same reason so I will let this PR sit until those errors are resolved.

Reply:
Thank you! Looks good, no requirement to add all translations, I think this can be handled later by us or other contributors 👌

Reply:
Thank you very much @arturi !